be/test/exprs/bitmap_functions_test.cpp

```diff
@@ -2588,6 +2698,116 @@ TEST_F(VecBitmapFunctionsTest, bitmap_subset_limit) {
     }
 }
 
+TEST_F(VecBitmapFunctionsTest, sub_bitmap_limit_special_cases) {
+    // empty bitmap
+    {
+        auto bitmap_column = BitmapColumn::create();
```

Comment:
create some cases where column is only null / const column

Reply:
it's been added in normal cases.